src/App.css

```
@@ -0,0 +1,1190 @@
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  box-shadow: 0 0 40px rgba(220, 238, 255, 0.6);
+  position: relative;
+}
+
+.breathing-circle.active {
+  animation: breatheGlow 8s ease-in-out infinite;
+}
+
+@keyframes breatheGlow {
```

Stylelint (17.2.0)

[error] Expected keyframe name "breatheGlow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)